pkg/daemon/ceph/osd/agent.go

```diff
@@ -571,6 +571,11 @@ func waitForRebalance(context *clusterd.Context, clusterName string, osdID int,
 					osdID, initialUsage, currUsage)
 			}
 
+			// when the initial OSD PG count is zero there is nothing to do here
+			if init.Pgs == "0" || init.UsedKB == "0" {
```

Comment:
we are comparing `json.Number` types to strings, is that supported and work as expected for `json.Number` comparisons?

Reply:
@jbw976 `json.Number` is type `string`, so yes that does a normal string to string comparison, otherwise go vet/fmt would tell use.